### PR TITLE
Fix \approx check for pre-0.5 versions

### PR DESCRIPTION
**Pull request by anriseth**
Ref @tkelman comment https://github.com/timholy/PositiveFactorizations.jl/pull/9#pullrequestreview-19172409

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -45,10 +45,11 @@ end
 
 A = [1 0; 0 -2]
 F = eigfact(Positive, A)
-if VERSION < v"v0.5-"
-    @test_approx_eq full(F) abs(A)
-else
+if isdefined(≈)
     @test full(F) ≈ abs.(A)
+
```
Comment (timholy) [inline test/runtests.jl]: can you delete this blank line? (sorry)

**Comment by anriseth** [comment]
Hmm. I don't understand how to fix this. From what I see, \approx was implemented in Julia commit 9ccdbffe, which has VERSION v0.4.0-dev. So @tkelman, what version of the v0.5 pre-releases did you mean that the test in #9 would fail on?

**Comment by tkelman** [comment]
it might not be the introduction of it, but some later change that made it more capable that you're relying on here

File: test/runtests.jl [hunk 2]
```diff
@@ -45,10 +45,10 @@ end
 
 A = [1 0; 0 -2]
 F = eigfact(Positive, A)
-if VERSION < v"v0.5-"
-    @test_approx_eq full(F) abs(A)
-else
+if isdefined(:≈)
     @test full(F) ≈ abs.(A)
```
Comment (tkelman) [inline test/runtests.jl]: looks like 0.4 is having problems with the `abs.(A)`, which using `@compat` would help with since that is a syntax rewrite

Comment (anriseth) [inline test/runtests.jl]: I didn't do this originally, due to https://github.com/JuliaLang/Compat.jl/issues/310. Fixed now

Comment (tkelman) [inline test/runtests.jl]: ah, right

**Comment by timholy** [comment]
Thanks!

**Pull request merged**